Gemfile

```diff
@@ -2,9 +2,10 @@ source 'https://rubygems.org'
 ruby '2.2.2'
 
 gem "csvlint"
+gem "haml_lint", "0.27.0"
 gem "puppet-lint", "2.1.1"
 gem "reek"
-gem "rubocop", "0.49.1"
+gem "rubocop", "0.51.0"
```

Comment:
@IpshitaC  I think you fixes the `Travis CI` problem https://github.com/coala/coala-bears/issues/2116

Reply:
@IpshitaC , can you create a separate PR for this change please ;-)